politeiawww/backend.go

```diff
@@ -450,6 +450,61 @@ func (b *backend) loadInventory() (*pd.InventoryReply, error) {
 	}, nil
 }
 
+func (b *backend) getProposals(after, before string, statusMap map[www.PropStatusT]bool) []www.ProposalRecord {
+	b.RLock()
+	defer b.RUnlock()
+
+	// pageStarted stores whether or not it's okay to start adding proposals
+	// to the array. If the after or before parameter is supplied, we must find
+	// the beginning (or end) of the page first.
+	pageStarted := (after == "" && before == "")
+	beforeIdx := -1
+	proposals := make([]www.ProposalRecord, 0)
+
+	// Iterate in reverse order because they're sorted by oldest timestamp first.
+	for i := len(b.inventory) - 1; i >= 0; i-- {
+		p := b.inventory[i]
+		if _, ok := statusMap[p.Status]; ok {
+			if pageStarted {
```

Comment:
switch (true) { in place of if-else chains (idiomatic)

Reply:
as a result the break needs to be changed to break "TAG Before the for like this 'tag:'".